packages/amber/views/Games/GamesDialog.tsx

```diff
@@ -351,32 +371,36 @@ export const GamesDialog: React.FC<GamesDialogProps> = ({ open, onClose, initial
                 inputProps={{ autoCapitalize: 'none' }}
               />
             </GridItem>
-            <GridItem xs={12} md={12}>
-              <p>
-                You are welcome to start and end the game at any time (within reason), but if the game overlaps two
-                slots, please enter two games and mark them as parts one and two.
-              </p>
-              {configuration.virtual && (
-                <p>Please keep in mind that you might have players from multiple time zones in your game.</p>
-              )}
-              <SelectField
-                name='estimatedLength'
-                label='Estimated Length'
-                margin='normal'
-                fullWidth
-                selectValues={estimatedLengthOptions}
-              />
-            </GridItem>
-            <GridItem xs={12} md={12}>
-              <SlotOptionsSelect
-                name='slotPreference'
-                label='Slot Preference'
-                year={values.year}
-                margin='normal'
-                required
-                fullWidth
-              />
-            </GridItem>
+            {!slim && (
+              <>
+                <GridItem xs={12} md={12}>
+                  <p>
+                    You are welcome to start and end the game at any time (within reason), but if the game overlaps two
```

Comment:
Looks like the logic and the text don't really make sense here, though fixing it is outside of the scope of this PR.  I.e. too much of the text doesn't really make sense outside of the virtual con. 

I'm take a look at cleaning it up for acnw later.

Reply:
Yeah, that's old text I bet. It seems to refer to estimated length and slot preference.

Similarly the first paragraph on the page on "Non-Amber Games" looks to be from the 90s.